www/css/index.css

```diff
@@ -6,15 +6,42 @@ a:active {
   display: none !important;
 }
 
+.navbar{
+  background-color: #eb7125 !important;
+  color: #fff;
+}
+
+.navbar .center{
+  font-weight: bold;
+}
+
 .tabbar .badge {
   background: #ffe0b3;
-  color: #eb7125
+  color: #eb7125;
+  border: 1px solid #eb7125;
+  position: absolute;
+  top: 0;
+  margin-left: 17px;
+  margin-top: 2px;
 }
 
-.tabbara.active .badge {
+.tabbar .active .badge {
   background: #fff;
 }
 
+.tabbar i{
+  font-size: 24px;
+}
+
+.tabbar i.fa-comments{
+  font-size: 30px;
+}
+
+.tabbar i.fa-home{
+  padding-top: 1px;
+  font-size: 32px;
```

Comment:
Is this icon much smaller than the others? Or why does it have to be 8px larger than the others